LiveChat/components/Chat.js

```diff
@@ -134,20 +40,25 @@ export default class Chat extends React.Component {
     if (this.props.isChatOn) {
       return (
         <View
-          animation="lightSpeedIn"
+          animation="fadeInUp"
           style={styles.container}
           ref={(ref) => { this.chat = ref; }}
         >
           <NavigationBar chatTitle={this.props.chatTitle} closeChat={this.closeChat} />
```

Comment:
Referencing to the previous comment:
```suggestion
          <NavigationBar chatTitle={this.props.chatTitle} closeChat={this.props.closeChat} />
```